modules/ROOT/partials/integrations/svelte-tech-ref.adoc

```diff
@@ -58,20 +58,30 @@ include::partial$misc/get-an-api-key.adoc[]
 
 {cloudname} License key.
 
-Use this option when self-hosting {productname} instead of loading from {cloudname}. For more information, see: xref:license-key.adoc[License Key].
+Use this option when self-hosting {productname} instead of loading from {cloudname}. License keys must start with the "T8LK:" prefix and use client-side JWT validation that runs entirely in the browser, requiring no server communication or "phone home" checks. For more information, see: xref:license-key.adoc[License Key].
```

Comment:
```suggestion
Use this option when self-hosting {productname} instead of loading from {cloudname}. For more information, see: xref:license-key.adoc[License Key].
```